### DISCUSSION TITLE
Hardcoded Group & Church generations

### DISCUSSION THREAD

**Discussion by yorkaspacher** [DiscipleTools/disciple-tools-theme#1592] [Q&A] [answered]
@corsacca This Group & Church Generations chart seems to be hard-coded to pull only the default options: Pre-Group, Group, Church. We have altered the options in this field, but it will not pull them. Any insight?
<img width="664" alt="image" src="https://user-images.githubusercontent.com/47872338/155134530-a345e8a3-6464-4b4c-a36f-b88bd1cd967a.png">

**Comment by corsacca** [comment, marked as answer]
Correct, this is currently hardcoded to only work with the default options.

**Reply by corsacca** [reply]
See created issue: https://github.com/DiscipleTools/disciple-tools-theme/issues/1600
